crates/sui/src/unit_tests/snapshots/sui__upgrade_compatibility__upgrade_compatibility_tests__enum.snap

```diff
@@ -0,0 +1,210 @@
+   │         ^ New unexpected variant 'D'.
+   │
+   = Enums are part of a module's public interface and cannot be changed during an upgrade.
+   = Restore the original enum's variants for enum 'EnumChangeAndAddVariant' including the ordering.
+
+error[Compatibility E01002]: type mismatch
+   ┌─ /Users/jordanjennings/code/sui/crates/sui/src/unit_tests/fixtures/upgrade_errors/enum_errors_v2/sources/UpgradeErrors.move:42:17
+   │
+42 │     public enum EnumChangeAndRemoveVariant {
+   │                 ^^^^^^^^^^^^^^^^^^^^^^^^^^ Missing variant 'C'.
+   │
+   = Enums are part of a module's public interface and cannot be changed during an upgrade.
+   = Restore the original enum's variant 'C' for enum 'EnumChangeAndRemoveVariant' including the ordering.
+
+error[Compatibility E01002]: type mismatch
+   ┌─ /Users/jordanjennings/code/sui/crates/sui/src/unit_tests/fixtures/upgrade_errors/enum_errors_v2/sources/UpgradeErrors.move:44:9
+   │
+42 │     public enum EnumChangeAndRemoveVariant {
+   │                 -------------------------- Enum definition
+43 │         A,
+44 │         C, // changed to C
+   │         ^ Mismatched variant name 'C', expected 'B'.
+   │
+   = Enums are part of a module's public interface and cannot be changed during an upgrade
+   = Restore the original enum's variants for enum 'EnumChangeAndRemoveVariant' including the ordering.
+
+error[Compatibility E01003]: ability mismatch
+   ┌─ /Users/jordanjennings/code/sui/crates/sui/src/unit_tests/fixtures/upgrade_errors/enum_errors_v2/sources/UpgradeErrors.move:49:17
+   │
+49 │     public enum EnumAddAbilityWithTypes has copy, drop { // drop added
+   │                 ^^^^^^^^^^^^^^^^^^^^^^^ Unexpected abilities 'drop'
+   │
+   = Enums are part of a module's public interface and cannot be changed during an upgrade.
+   = Restore the original enum's abilities for enum 'EnumAddAbilityWithTypes' including the ordering.
+
+error[Compatibility E01003]: ability mismatch
+   ┌─ /Users/jordanjennings/code/sui/crates/sui/src/unit_tests/fixtures/upgrade_errors/enum_errors_v2/sources/UpgradeErrors.move:53:17
+   │
+53 │     public enum EnumRemoveAbilityWithTypes has copy { // drop removed
+   │                 ^^^^^^^^^^^^^^^^^^^^^^^^^^ Missing abilities 'drop'
+   │
+   = Enums are part of a module's public interface and cannot be changed during an upgrade.
+   = Restore the original enum's abilities for enum 'EnumRemoveAbilityWithTypes' including the ordering.
+
+error[Compatibility E01002]: type mismatch
+   ┌─ /Users/jordanjennings/code/sui/crates/sui/src/unit_tests/fixtures/upgrade_errors/enum_errors_v2/sources/UpgradeErrors.move:59:9
+   │
+57 │     public enum EnumAddVariantWithTypes {
+   │                 ----------------------- Enum definition
+58 │         A { a: u8 },
+59 │         B { b: u8 }, // added
+   │         ^^^^^^^^^^^ New unexpected variant 'B'.
+   │
+   = Enums are part of a module's public interface and cannot be changed during an upgrade.
```

Comment:
I'm seeing some weird characters at the end of these lines on GitHub. Any idea what is up here?

Reply:
the utf8 character code for the period is `0x2e` which is a regular period 🤷 guessing highlighting feature in github is having an issue, single quotes similarly are also throwing it off.
`cat -e` seems to suggest the newlines are fine.
<img width="1068" alt="image" src="https://github.com/user-attachments/assets/23af46ea-1edf-4e11-a53d-336cb6f12fcc">